src/sbt-slicer.cpp

```diff
@@ -144,6 +97,14 @@ llvm::cl::opt<bool> memsafety(
                 "by the given slicing criteria."),
         llvm::cl::init(false), llvm::cl::cat(SlicingOpts));
 
+llvm::cl::opt<bool> cutoff_diverging(
```

Comment:
Wasn't this option moved to `options`? (Line 252 uses `options.cutoffDiverging`)

Reply:
Sorry!

Reply:
Resolved in bf87d5a.